src/info/InfoMappings.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { Namespace } from '../search/types/searchTypes';
+import UniProtKBInfo from './UniProtKBInfo';
+
+const infoMappings: {
+  [index in Namespace]: {
+    name: string;
+    info: JSX.Element;
+    links: Array<{ title: string; destination: string }>;
```

Comment:
I don't have a solid proof for this, but I think the so called angle-bracket type definitions are discouraged in type script e.g.  `Array<...>`

I'm using a TSLint configuration called [Airbnb TS config](https://github.com/progre/tslint-config-airbnb/#readme), but it's not anything official from Airbnb, but it does have a good mix of community accepted rules. I think Microsoft coding-styles also discourages this type of type definitions, except for Generic and Iterator types.

I think it's worth look into this more in depth.

Reply:
I *think* they are only discouraged when it's a single type, e.g. `Array<string>` instead of `string[]`.

Reply:
Maybe we should dig deeper into this and have a chat about it later. To me, the TypeScript convention seems to be:

    type Link = { title: string; destination: string };
    links: Link[];    // and later in your code...